sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <full-page-container>
+    <h2 class="color-blue">Search Institution</h2>
+    <v-row class="mt-5">
+      <v-col><label for="legalName">Legal Name</label></v-col>
+      <v-col><label for="operatingName">Operating Name</label></v-col>
+      <v-col></v-col>
+      <v-col></v-col>
+    </v-row>
+    <v-row>
+      <v-col><InputText type="text" v-model="legalName"/></v-col>
+      <v-col><InputText type="text" v-model="operatingName"/></v-col>
+      <v-col></v-col>
+      <v-col
+        ><v-btn
+          :disabled="!legalName && !operatingName"
+          color="primary"
+          class="p-button-raised"
+          @click="searchInstitutions()"
+        >
+          <v-icon size="25" class="mr-2">mdi-account-outline</v-icon>
+          Search
+        </v-btn>
+      </v-col>
+    </v-row>
+
+    <DataTable
+      v-if="institutionsFound"
+      class="mt-4"
+      :autoLayout="true"
+      :value="institutions"
+    >
+      <Column field="legalName" header="Legal Name" :sortable="true">
+        <template #body="slotProps">
+          <div class="p-text-capitalize">
+            {{ slotProps.data.legalName }}
+          </div>
+        </template>
+      </Column>
+      <Column field="operatingName" header="Operating Name" :sortable="true">
+        <template #body="slotProps">
+          <div class="p-text-capitalize">
+            {{ slotProps.data.operatingName }}
+          </div>
+        </template>
+      </Column>
+      <Column field="address" header="Address">
+        <template #body="slotProps">
+          <span class="mt-2">
```

Comment:
Can we create a component to display the address in the same way all around the application?
If we create a common method to format it we can reuse it even on form.io forms.